perspective-component/web/build.gradle.kts

```diff
@@ -89,14 +98,17 @@ task deepClean() {
         delete(file("node_modules"))
     }
 
-    dependsOn clean
+    dependsOn(project.tasks.named("clean"))
 }
 
-
 // make sure the gateway project doesn't process resources until the webpack task is done.
-project(":gateway") {
-    tasks.getByName("processResources").dependsOn(webpack)
+project(":gateway")?.tasks?.named("processResources")?.configure {
```

Comment:
typesafe accessor possible?

Reply:
Not currently - right now they seem to only function in specific areas and don't return the _Project_ (which has the _tasks_ ), but return some sort of delegate, and only in certain scopes (like _dependencies_)